etc/schema.yaml

```diff
@@ -133,6 +135,7 @@ samples:
     std_dev_insert_size: {      type: 'float' }
     median_insert_size: {       type: 'integer' }
     median_abs_dev_insert_size: { type: 'integer' }
+    other_insert_type: {        type: 'dict', required: False }
```

Comment:
If we're switching to `non_fr_inserts`, we'd need to change this as well

Reply:
you're too quick to review.
Didn't have time to push.